google/cloud/aiplatform/metadata/schema/google/artifact_schema.py

```diff
@@ -268,3 +268,251 @@ def __init__(
             metadata=extended_metadata,
             state=state,
         )
+
+
+class ClassificationMetrics(base_artifact.BaseArtifactSchema):
+    """A Google artifact representing evaluation Classification Metrics."""
+
+    schema_title = "google.ClassificationMetrics"
+
+    def __init__(
+        self,
+        *,
+        au_prc: Optional[float] = None,
+        au_roc: Optional[float] = None,
+        log_loss: Optional[float] = None,
+        artifact_id: Optional[str] = None,
+        uri: Optional[str] = None,
+        display_name: Optional[str] = None,
+        schema_version: Optional[str] = None,
+        description: Optional[str] = None,
+        metadata: Optional[Dict] = None,
+        state: Optional[gca_artifact.Artifact.State] = gca_artifact.Artifact.State.LIVE,
+    ):
+        """Args:
+        au_prc (float):
+            Optional. The Area Under Precision-Recall Curve metric.
+            Micro-averaged for the overall evaluation.
+        au_roc (float):
+            Optional. The Area Under Receiver Operating Characteristic curve metric.
+            Micro-averaged for the overall evaluation.
+        log_loss (float): 
+            Optional. The Log Loss metric.
+        artifact_id (str):
+            Optional. The <resource_id> portion of the Artifact name with
+            the format. This is globally unique in a metadataStore:
+            projects/123/locations/us-central1/metadataStores/<metadata_store_id>/artifacts/<resource_id>.
+        uri (str):
+            Optional. The uniform resource identifier of the artifact file. May be empty if there is no actual
+            artifact file.
+        display_name (str):
+            Optional. The user-defined name of the Artifact.
+        schema_version (str):
+            Optional. schema_version specifies the version used by the Artifact.
+            If not set, defaults to use the latest version.
+        description (str):
+            Optional. Describes the purpose of the Artifact to be created.
+        metadata (Dict):
+            Optional. Contains the metadata information that will be stored in the Artifact.
+        state (google.cloud.gapic.types.Artifact.State):
+            Optional. The state of this Artifact. This is a
+            property of the Artifact, and does not imply or
+            capture any ongoing process. This property is
+            managed by clients (such as Vertex AI
+            Pipelines), and the system does not prescribe or
+            check the validity of state transitions.
+        """
+        extended_metadata = copy.deepcopy(metadata) if metadata else {}
+        if au_prc is not None:
```

Comment:
nit:  could simplify to  `if au_prc: ....` the general guidance is ```Use the "implicit" false if possible, e.g., if foo: rather than if foo != []:```

same for cases below.

Reply:
Fixed!